diff --git a/Config.in b/Config.in
index 6cb6b83dfb..3210af928c 100644
--- a/Config.in
+++ b/Config.in
@@ -2,7 +2,7 @@
 #
 # Copyright (C) 2006-2013 OpenWrt.org
 
-mainmenu "LibWrt Configuration"
+mainmenu "OpenWrt Configuration"
 
 config MODULES
 	modules
diff --git a/config/Config-images.in b/config/Config-images.in
index 4f916f386f..853bdf410f 100644
--- a/config/Config-images.in
+++ b/config/Config-images.in
@@ -280,7 +280,7 @@ menu "Target Images"
 	config GRUB_TITLE
 		string "Title for the menu entry in GRUB"
 		depends on GRUB_IMAGES || GRUB_EFI_IMAGES
-		default "LibWrt"
+		default "OpenWrt"
 		help
 		  This is the title of the GRUB menu entry.
 		  If unspecified, it defaults to OpenWrt.
diff --git a/include/version.mk b/include/version.mk
index 63ba4c6a42..6d1e795f76 100644
--- a/include/version.mk
+++ b/include/version.mk
@@ -36,11 +36,11 @@ VERSION_REPO:=$(call qstrip,$(CONFIG_VERSION_REPO))
 VERSION_REPO:=$(if $(VERSION_REPO),$(VERSION_REPO),https://downloads.immortalwrt.org/snapshots)
 
 VERSION_DIST:=$(call qstrip,$(CONFIG_VERSION_DIST))
-VERSION_DIST:=$(if $(VERSION_DIST),$(VERSION_DIST),LibWrt)
+VERSION_DIST:=$(if $(VERSION_DIST),$(VERSION_DIST),OpenWrt)
 VERSION_DIST_SANITIZED:=$(call sanitize,$(VERSION_DIST))
 
 VERSION_MANUFACTURER:=$(call qstrip,$(CONFIG_VERSION_MANUFACTURER))
-VERSION_MANUFACTURER:=$(if $(VERSION_MANUFACTURER),$(VERSION_MANUFACTURER),LibWrt)
+VERSION_MANUFACTURER:=$(if $(VERSION_MANUFACTURER),$(VERSION_MANUFACTURER),OpenWrt)
 
 VERSION_MANUFACTURER_URL:=$(call qstrip,$(CONFIG_VERSION_MANUFACTURER_URL))
 VERSION_MANUFACTURER_URL:=$(if $(VERSION_MANUFACTURER_URL),$(VERSION_MANUFACTURER_URL),https://immortalwrt.org/)
diff --git a/package/base-files/files/bin/config_generate b/package/base-files/files/bin/config_generate
index 91401dab86..58a7f4761a 100755
--- a/package/base-files/files/bin/config_generate
+++ b/package/base-files/files/bin/config_generate
@@ -312,7 +312,7 @@ generate_static_system() {
 	uci -q batch <<-EOF
 		delete system.@system[0]
 		add system system
-		set system.@system[-1].hostname='LibWrt'
+		set system.@system[-1].hostname='OpenWrt'
 		set system.@system[-1].timezone='CST-8'
 		set system.@system[-1].zonename='Asia/Shanghai'
 		set system.@system[-1].ttylogin='0'
diff --git a/package/base-files/files/etc/banner b/package/base-files/files/etc/banner
index fd0297f4d1..f3af3c014f 100644
--- a/package/base-files/files/etc/banner
+++ b/package/base-files/files/etc/banner
@@ -1,10 +1,8 @@
- __           ____    Commercial         __      
-/\ \       __/\  _`\       Prohibition  /\ \__   
-\ \ \     /\_\ \ \L\ \  __  __  __  _ __\ \ ,_\  
- \ \ \  __\/\ \ \  _ <'/\ \/\ \/\ \/\`'__\ \ \/  
-  \ \ \L\ \\ \ \ \ \L\ \ \ \_/ \_/ \ \ \/ \ \ \_ 
-   \ \____/ \ \_\ \____/\ \___x___/'\ \_\  \ \__\
-    \/___/   \/_/\/___/  \/__//__/   \/_/   \/__/                         
+  _______                     ________        __
+ |       |.-----.-----.-----.|  |  |  |.----.|  |_
+ |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
+ |_______||   __|_____|__|__||________||__|  |____|
+          |__| W I R E L E S S   F R E E D O M
  -----------------------------------------------------
-  %D %V, %C
+ %D %V, %C
  -----------------------------------------------------
diff --git a/package/base-files/image-config.in b/package/base-files/image-config.in
index acc570927d..281580c723 100644
--- a/package/base-files/image-config.in
+++ b/package/base-files/image-config.in
@@ -165,10 +165,10 @@ if VERSIONOPT
 	config VERSION_DIST
 		string
 		prompt "Release distribution"
-		default "LibWrt"
+		default "OpenWrt"
 		help
 			This is the name of the release distribution.
-			If unspecified, it defaults to LibWrt.
+			If unspecified, it defaults to OpenWrt.
 
 	config VERSION_NUMBER
 		string
diff --git a/package/network/config/wifi-scripts/files/lib/wifi/mac80211.uc b/package/network/config/wifi-scripts/files/lib/wifi/mac80211.uc
index 1ca3f61544..532ee03f12 100644
--- a/package/network/config/wifi-scripts/files/lib/wifi/mac80211.uc
+++ b/package/network/config/wifi-scripts/files/lib/wifi/mac80211.uc
@@ -104,7 +104,7 @@ set ${si}=wifi-iface
 set ${si}.device='${name}'
 set ${si}.network='lan'
 set ${si}.mode='ap'
-set ${si}.ssid='${defaults?.ssid || "LibWrt"}'
+set ${si}.ssid='${defaults?.ssid || "OpenWrt"}'
 set ${si}.encryption='${defaults?.encryption || "none"}'
 set ${si}.key='${defaults?.key || ""}'
 
